### PR TITLE
Clarify that Mac works with cuDNN5.

### PR DESCRIPTION
**Pull request by yifeif**

### CODE REVIEW

File: tensorflow/g3doc/get_started/os_setup.md [hunk 1]
```diff
@@ -580,8 +581,8 @@ Install the toolkit into e.g. `/usr/local/cuda`
 
 https://developer.nvidia.com/cudnn
 
-Download cuDNN v4 (v5 is currently a release candidate and is only supported when
-installing TensorFlow from sources).
+Download cuDNN v4 (v5 is only supported when
+installing TensorFlow from sources). If setting up on Mac, download cnDNN v5.
```
Comment (vrv) [inline tensorflow/g3doc/get_started/os_setup.md]: s/cnDNN/cuDNN

File: tensorflow/g3doc/get_started/os_setup.md [hunk 2]
```diff
@@ -580,8 +581,8 @@ Install the toolkit into e.g. `/usr/local/cuda`
 
 https://developer.nvidia.com/cudnn
 
-Download cuDNN v4 (v5 is currently a release candidate and is only supported when
-installing TensorFlow from sources).
+Download cuDNN v4 (v5 is only supported when
```
Comment (vrv) [inline tensorflow/g3doc/get_started/os_setup.md]: I don't think this is true anymore, I think we can recommend v5 everywhere

File: tensorflow/g3doc/get_started/os_setup.md [hunk 3]
```diff
@@ -7,11 +7,12 @@ github source.
 
 The TensorFlow Python API supports Python 2.7 and Python 3.3+.
 
-The GPU version (Linux only) works best with Cuda Toolkit 7.5 and
-cuDNN v4.  other versions are supported (Cuda toolkit >= 7.0 and
-cuDNN 6.5(v2), 7.0(v3), v5) only when installing from sources.
+The GPU version works best with Cuda Toolkit 7.5 and
+cuDNN v4 on Linux.  Other versions are supported (Cuda toolkit >= 7.0 and
```
Comment (vrv) [inline tensorflow/g3doc/get_started/os_setup.md]: I'd change this to v5, and remove v2 on the line below, since we recently removed cudnnV2 support.

**Comment by yifeif** [comment]
Keeping unix binary installation recommendation to v4 as we are still using v4 for this release. Removing v2 as suggested by Vijay. 
@vrv could you take another look? I would like to get this updated so people stop running into #3826 . Thanks!

**Comment by vrv** [comment]
LGTM

We can make the same change but with the v4->v5 changes in master :)

**Pull request merged**